spec/amp-cache-debugging.md

```diff
@@ -0,0 +1,28 @@
+# AMP Cache Debugging
+
+## Why is my doc broken on an AMP cache?
+
+Valid AMP documents typically appear and behave the same on AMP Caches as they
+do on the origin. However, there are some components and server configurations
+that can be problematic.
+
+If a particular document appears and behaves as expected on your origin, but not
+when viewed via the cache ([how to map origin URLs to Google's AMP
+Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)),
+try the following:
+
+1. Open your browser's developer/error tools console, and resolve
+any errors or warnings that appear.
+1. Run the document through [AMPBench](https://ampbench.appspot.com/) and
+resolve any unexpected errors or warnings.
+
+If you still have a problem after following these steps, check the table below.
+
+|Symptom|Issue|Solution|
+|---|---|---|
+|Web fonts do not appear (fallback fonts are used)|Cache not whitelisted by font provider|Contact font provider and ask them to whitelist [all caches](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md#cors-security-in-amp)|
+|Assets (e.g. fonts and images) do not appear #1 (**HTTP origins only**)|Document uses protocol-relative URLs|Switch to absolute URLs (that is, `http://www.site.com/doc/amp`, not `//www.site.com/doc/amp`)|
```

Comment:
@SchnWalter This item was specifically about HTTP origins, so `http` is correct. Probably `example.com` would be better but would really like to get this merged and so want to avoid making minor changes! (There's also another change I want to make to this doc soon (to mention `https://cdn.ampproject.org/caches.json`)--I can make the `example.com` change at the same time.)